tasks/buildpackages/make-deb.sh

```diff
@@ -64,22 +65,44 @@ function build_package() {
     # included in the distribution.
     #
     git clean -qdxff
-    #
-    # creating the distribution tarbal requires some configure
-    # options (otherwise parts of the source tree will be left out).
-    #
-    ./autogen.sh
-    # Building with LTTNG on Ubuntu Precise is not possible.
-    # It fails the LTTNG-is-sane check (it misses headers)
-    # And the Debian rules files leave it out anyway
-    case $codename in
-	precise) lttng_opt="--without-lttng" ;;
-	*) lttng_opt="--with-lttng" ;;
-    esac
-    ./configure $(flavor2configure $flavor) \
-        --with-rocksdb --with-ocf \
-        --with-nss --with-debug --enable-cephfs-java \
-        $lttng_opt --with-babeltrace
+
+    # autotools only works in jewel and below
+    if [[ "$majorvers" -le 10 ]] ; then
+        #
+        # creating the distribution tarbal requires some configure
+        # options (otherwise parts of the source tree will be left out).
+        #
+        ./autogen.sh
+        # Building with LTTNG on Ubuntu Precise is not possible.
+        # It fails the LTTNG-is-sane check (it misses headers)
+        # And the Debian rules files leave it out anyway
+        case $codename in
+	    precise) lttng_opt="--without-lttng" ;;
+	    *) lttng_opt="--with-lttng" ;;
+        esac
+        ./configure $(flavor2configure $flavor) \
+            --with-rocksdb --with-ocf \
+            --with-nss --with-debug --enable-cephfs-java \
+            $lttng_opt --with-babeltrace
+    else
+        mkdir -p obj-x86_64-linux-gnu
+        cd obj-x86_64-linux-gnu 
+        cmake .. \
+            -DCMAKE_INSTALL_PREFIX=/usr \
+            -DCMAKE_INSTALL_LIBDIR=/usr/lib64 \
+            -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib \
+            -DCMAKE_INSTALL_LOCALSTATEDIR=/var \
+            -DCMAKE_INSTALL_SYSCONFDIR=/etc \
+            -DCMAKE_INSTALL_MANDIR=/usr/share/man \
+            -DCMAKE_INSTALL_DOCDIR=/usr/share/doc/ceph \
+            -DWITH_MANPAGE=ON \
+            -DWITH_SYSTEMD=ON \
+            -DWITH_CEPHFS_JAVA=ON \
+            -DWITH_SELINUX=ON \
+            -DWITH_LTTNG=ON \
+            -DHAVE_BABELTRACE=ON \
+            -DALLOCATOR=tcmalloc
+    fi
```

Comment:
we should just check the existence of `make-dist`. use it if it exists, and `make dist` otherwise.